.github/workflows/automated-a11y.yml

```diff
@@ -32,16 +32,47 @@ jobs:
       uses: actions/setup-node@v4
       with:
         node-version: ${{ matrix.node-version }}
-    
+
+    - name: Cache node modules
+      id: node-modules-cache
+      uses: actions/cache@v4
+      env:
+        cache-name: node-modules-v1
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-cache-${{ env.cache-name }}-node-${{ matrix.node-version }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-cache-${{ env.cache-name }}-node-${{ matrix.node-version }}-yarn-
+
+    - name: Enable Corepack
+      run: corepack enable
+
     - name: Install Project Dependencies
-      run: yarn install
-    
+      if: steps.node-modules-cache.outputs.cache-hit != 'true'
+      run: yarn install --immutable
+
+    - name: Extract Playwright version
+      id: playwright-version
+      run: echo "PLAYWRIGHT_VERSION=$(jq -r '.devDependencies["@playwright/test"]' packages/e2e-playwright/package.json)" >> $GITHUB_ENV
+
+    - name: Cache Playwright
+      # Using a cache for the webkit binary files causes this error on subsequent runs
```

Comment:
Also what is the difference in performance we get here, can you make the comparison? 

https://playwright.dev/docs/ci#caching-browsers

Reply:
| E2E Chromium, API v71 (with cache)   | E2E Chromium, API v71 (without cache) |
| -------- | ------- |
| 8m 7s  | 9m 36s    |

| E2E Firefox, API v71 (with cache)   | E2E Firefox, API v71 (without cache) |
| -------- | ------- |
| 7m 35s  | 10m 56s   |

We get about a 1 to 2 minute reduction in time it takes for the tests on Firefox and Chromium which are currently the only tests which cached playwright browsers.